### PR TITLE
feat(connectors): install connectors into all bundles except none

### PR DESCRIPTION
**Pull request by portellaa**

### CODE REVIEW

File: .github/workflows/prereleased.yaml [hunk 1]
```diff
@@ -137,6 +137,10 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Add authentication to requirements file
+      run: |
+        sed -i 's/https:\/\/pypi.ydata.ai/https:\/\/${{ secrets.PRIVATE_PYPI_USERNAME }}:${{ secrets.PRIVATE_PYPI_PASSWORD }}@pypi.ydata.ai/g' data-science/visualcode_ydata/requirements.txt
```
Comment (UrbanoFonseca) [inline .github/workflows/prereleased.yaml:142]: targets are misaligned to `visualcode_ydata`

File: .github/workflows/prereleased.yaml [hunk 2]
```diff
@@ -707,6 +751,10 @@ jobs:
...
```
Comment (UrbanoFonseca) [inline .github/workflows/prereleased.yaml:756]: is this missing an update to visualcode-torch?

File: .github/workflows/prereleased.yaml [hunk 3]
```diff
@@ -653,6 +685,10 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Add authentication to requirements file
+      run: |
+        sed -i 's/https:\/\/pypi.ydata.ai/https:\/\/${{ secrets.PRIVATE_PYPI_USERNAME }}:${{ secrets.PRIVATE_PYPI_PASSWORD }}@pypi.ydata.ai/g' data-science/visualcode_tensorflow-2./requirements.txt
```
[the remark this answers is not in the document]
Comment (portellaa) [reply .github/workflows/prereleased.yaml]: yeap 🤦

File: .github/workflows/prereleased.yaml (hunk 2 above)
Comment (portellaa) [reply .github/workflows/prereleased.yaml]: yes 🤦  i must be sleeping 🤦

File: data-science/jupyterlab_python_tensorflow-1.15/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+--extra-index-url https://azory:arHWmo3jr-ca.aP1@pypi.ydata.ai
```
[the remark this answers is not in the document]
Comment (portellaa) [reply data-science/jupyterlab_python_tensorflow-1.15/requirements.txt]: this was just for testing purposes

[the remark this answers is not in the document]
Comment (portellaa) [reply data-science/jupyterlab_python_tensorflow-1.15/requirements.txt]: thanks

File: data-science/jupyterlab_python_tensorflow-2.3/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+--extra-index-url https://azory:arHWmo3jr-ca.aP1@pypi.ydata.ai
```
Comment (UrbanoFonseca) [inline data-science/jupyterlab_python_tensorflow-2.3/requirements.txt:1]: replace to standard pypi

File: data-science/visualcode_tensorflow-1.15/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+--extra-index-url https://azory:arHWmo3jr-ca.aP1@pypi.ydata.ai
```
Comment (UrbanoFonseca) [inline data-science/visualcode_tensorflow-1.15/requirements.txt:1]: replace to standard pypi

File: data-science/visualcode_tensorflow-2.3/requirements.txt [hunk 7]
```diff
@@ -0,0 +1,8 @@
+--extra-index-url https://azory:arHWmo3jr-ca.aP1@pypi.ydata.ai
```
Comment (UrbanoFonseca) [inline data-science/visualcode_tensorflow-2.3/requirements.txt:1]: replace to standard pypi

File: data-science/visualcode_torch-1.7/requirements.txt [hunk 8]
```diff
@@ -0,0 +1,8 @@
+--extra-index-url https://azory:arHWmo3jr-ca.aP1@pypi.ydata.ai
```
Comment (UrbanoFonseca) [inline data-science/visualcode_torch-1.7/requirements.txt:1]: replace to standard pypi

**Review by UrbanoFonseca — APPROVED** [review]

**Pull request merged**